leadbutt.py

```diff
@@ -144,18 +162,19 @@ def leadbutt(config_file, cli_options, verbose=False, **kwargs):
             # we need a copy of the metric dict with the MetricName swapped out
             this_metric = metric.copy()
             this_metric['MetricName'] = metric_name
-            results = conn.get_metric_statistics(
-                period_local,  # minimum: 60
-                start_time,
-                end_time,
-                metric_name,  # RequestCount, CPUUtilization
-                metric['Namespace'],  # AWS/ELB, AWS/EC2
-                metric['Statistics'],  # Sum, Maximum
+            results = get_metric_statistics(
+                connection=conn,
+                period=period_local,
+                start_time=start_time,
+                end_time=end_time,
+                metric_name=metric_name,
+                namespace=metric['Namespace'],
+                statistics=metric['Statistics'],
                 dimensions=metric['Dimensions'],
-                unit=unit,
+                unit=unit
             )
-            # sys.stderr.write('{} {}\n'.format(options['Count'], len(results)))
             output_results(results, this_metric, options)
+            time.sleep(float(kwargs.get('interval', 0)) / 1000.0)
```

Comment:
There is a comment in the docstring (https://github.com/krux/cloudwatch-to-graphite/pull/7/files#diff-75d44ccbc6fb95936c59f2fb20cbb35cR9), which ... doubles as the options definition.